nvflare/fuel/hci/client/file_transfer.py

```diff
@@ -191,23 +189,17 @@ def get_spec(self):
                     usage="download_binary file_name ...",
                     handler_func=self.download_binary_file,
                 ),
-                CommandSpec(
-                    name=self.upload_folder_cmd_name,
-                    description="upload application to the server",
-                    usage=self.upload_folder_cmd_name + " application_folder",
-                    handler_func=self.upload_folder,
-                ),
                 CommandSpec(
                     name="download_folder",
                     description="download a folder from the server",
                     usage="download_folder folder_name",
                     handler_func=self.download_folder,
                 ),
                 CommandSpec(
-                    name="upload_job",
-                    description="upload application to the server",
-                    usage="upload_job job_folder",
-                    handler_func=self.upload_job,
+                    name="submit_job",
```

Comment:
submit_job is repeated multiple times, should we create a constant variable ?

Reply:
Yes. All command names are repeated multiple times. We need to do a refactoring.